compiler/src/dmd/statementsem.d

```diff
@@ -3444,10 +3445,7 @@ Statement statementSemanticVisit(Statement s, Scope* sc)
             // https://issues.dlang.org/show_bug.cgi?id=23159
             if (!global.params.useExceptions)
             {
-                version (IN_GCC)
-                    error(oss.loc, "`%s` cannot be used with `-fno-exceptions`", Token.toChars(oss.tok));
-                else
-                    error(oss.loc, "`%s` cannot be used with -betterC", Token.toChars(oss.tok));
+                error(oss.loc, "`%s` cannot be used with `-betterC`", Token.toChars(oss.tok));
```

Comment:
That's a change for GDC, should use `SwitchExceptions`.

Reply:
good catch